scene/main/node.h

```diff
@@ -228,6 +228,8 @@ class Node : public Object {
 		NOTIFICATION_TRANSLATION_CHANGED = 24,
 		NOTIFICATION_INTERNAL_PROCESS = 25,
 		NOTIFICATION_INTERNAL_PHYSICS_PROCESS = 26,
+		NOTIFICATION_SCROLL_BEGIN = 27,
```

Comment:
This should most likely go in Control notifications instead of Node, since only controls will use it.

Other than that, PR is perfect.